### PR TITLE
fix(dom_adapter): remove logError from logGroup

### LINKED ISSUE #12923: Router throws Illegal Invocation error when enableTracing is true
**I'm submitting a ...
[X] bug report => search github for a similar issue or PR before submitting

**Current behavior**

Set the router enableTracing option to true.  As soon as hte app loads and begins to route and Illegal Invocation error is thrown.

It did this with a brand new app created with angular-cli.  I added one test component and create a default route for it and it loaded fine.  Then I set enableTracing to true and it started throwing the error.

**Expected behavior**
No error thrown, just shows the router trace

This is occurring in a NG 2.1.0 (router 3.1.0) and above.  I found it while upgrading an app from 2.0.2 (where it worked fine) to 2.2.0.  

Here's a link to the error that's thrown: [error.png](https://postimg.org/image/dh20gkwct/)

### PR DESCRIPTION
**Pull request by DzmitryShylovich**
Fixes https://github.com/angular/angular/issues/12923

### CODE REVIEW

**Comment by DzmitryShylovich** [comment]
btw why do we call [logError](https://github.com/DzmitryShylovich/angular/blob/8615908fefe0315c6f86414d528a9d8bb320a372/modules/%40angular/platform-browser/src/browser/browser_adapter.ts#L95) inside `logGroup`  ?

File: modules/@angular/platform-browser/src/browser/browser_adapter.ts [hunk 1]
```diff
@@ -79,7 +79,7 @@ export class BrowserDomAdapter extends GenericBrowserDomAdapter {
   // TODO(tbosch): move this into a separate environment class once we have it
   logError(error: string): void {
     if (window.console) {
-      (window.console.error || window.console.log)(error);
```
Comment (vicb) [inline modules/@angular/platform-browser/src/browser/browser_adapter.ts]: the behavior is not the same here (former is try error then log)

File: modules/@angular/platform-browser/src/browser/browser_adapter.ts [hunk 2]
```diff
@@ -92,7 +92,6 @@ export class BrowserDomAdapter extends GenericBrowserDomAdapter {
   logGroup(error: string): void {
     if (window.console) {
       window.console.group && window.console.group(error);
-      this.logError(error);
```
Comment (vicb) [inline modules/@angular/platform-browser/src/browser/browser_adapter.ts]: probably reasonable, have you check the call sites to see if we do not rely on this printing an `error` ?

Comment (DzmitryShylovich) [inline modules/@angular/platform-browser/src/browser/browser_adapter.ts]: https://github.com/angular/angular/search?utf8=%E2%9C%93&q=logGroup

File: modules/@angular/platform-browser/src/browser/browser_adapter.ts (hunk 1 above)
Comment (DzmitryShylovich) [inline modules/@angular/platform-browser/src/browser/browser_adapter.ts]: fixed

**Pull request merged**